atomicapp/external/templates/nulecule/Dockerfile.tpl

```diff
@@ -0,0 +1,10 @@
+FROM projectatomic/atomicapp:0.4.1
+
+MAINTAINER Your Name <you@email.com>
+
+LABEL io.projectatomic.nulecule.specversion="0.0.2" \
+      io.projectatomic.nulecule.providers="kubernetes, docker" \
+      Build="docker build --rm --tag test/$app_name-atomicapp ."
```

Comment:
what is this? i've never seen this before. this shouldn't be here as it's not common convention to have it.